src/homeWork7/Animals.java

```diff
@@ -0,0 +1,20 @@
+package homeWork7;
+
+public abstract class Animals {
+
+    private static int counter;
+    public String name;
+
+    public Animals(String name) {
+        this.name = name;
+        counter++;
+    }
+
+    public abstract void run (int length);
+
+    public abstract void swim (int length);
+
+    public int getCounterAnimals(){
```

Comment:
Would be better to make this method static. Imagine a situation when you haven't created any cat yet, but you want to call a `getCounterAnimals` method. To do that, you are supposed to create a `new Animal()`. This is very inflexible. Would be better to use a static method to check your animal counter anytime without having a dependency on a specific object, like this `Animal.getAnimalCounter()`